ban/core/versioning.py

```diff
@@ -242,6 +250,7 @@ class Diff(db.Model):
     old = db.ForeignKeyField(Version, null=True)
     # new is empty after delete.
     new = db.ForeignKeyField(Version, null=True)
+    insee = db.CharField(length=5)
```

Comment:
I thought that you have a perf issue so I proposed to denormalize but re-reading your comment that doesn't look to be the case. Forget about it and sorry for the noise 😃